testing/run_tests.py

```diff
@@ -1209,20 +1196,17 @@ def main():
     )
 
   # Use this type to exclusively run impeller vulkan tests.
-  # TODO (https://github.com/flutter/flutter/issues/113961): Remove this once
-  # impeller vulkan tests are stable.
   if 'impeller-vulkan' in types:
     build_name = args.variant
     try:
       xvfb.start_virtual_x(build_name, build_dir)
-      vulkan_gtest_filter = parse_impeller_vulkan_filter()
-      gtest_flags = shuffle_flags
-      gtest_flags.append(vulkan_gtest_filter)
       run_engine_executable(
           build_dir,
           'impeller_unittests',
           engine_filter,
-          gtest_flags,
+          shuffle_flags + [
+              '--gtest_filter=-*/OpenGLES:Play/TypographerTest.MaybeHasOverlapping/Vulkan:Play/TypographerTest.GlyphAtlasWithLotsOfdUniqueGlyphSize/Vulkan',
```

Comment:
I'll create follow up issues for these 2 tests.

Reply:
SGTM. Could you make a TODO here that links to the issue?

Reply:
Done.